ros2_control_test_assets/include/ros2_control_test_assets/descriptions.hpp

```diff
@@ -73,6 +127,21 @@ const auto urdf_head =
     <parent link="link2"/>
     <child link="tool_link"/>
   </joint>
+  <link name="tool_link">
+    </link>
+  <gazebo reference="base_link">
+    <material>Gazebo/Gray</material>
+  </gazebo>
+  <gazebo reference="link1">
+    <material>Gazebo/Gray</material>
+  </gazebo>
+  <gazebo reference="link2">
+    <material>Gazebo/Gray</material>
+  </gazebo>
+  <gazebo>
+    <plugin filename="libgazebo_ros_control.so" name="gazebo_ros_control">
+      </plugin>
+  </gazebo>
```

Comment:
Why do we need gazebo here?

Reply:
I just copy this from the test file and use a more comprehensive one. But I can remove this. since it is not relevant here.